tests/bundle/bundle-test.ts

```diff
@@ -267,25 +158,21 @@ async function runAll() {
   const credential = await Issuer.createCredential({
     cType: DriversLicense,
     claims: content,
-    subject: bob.id,
-    issuer: alice.id,
+    subject: bob.did,
+    issuer: alice.did,
   })
 
   console.info('Credential subject conforms to CType')
 
   if (
     credential.credentialSubject.name !== content.name ||
     credential.credentialSubject.age !== content.age ||
-    credential.credentialSubject.id !== bob.id
+    credential.credentialSubject.id !== bob.did
   ) {
     throw new Error('Claim content inside Credential mismatching')
   }
 
-  const issued = await Issuer.issue(credential, {
-    did: alice.id,
-    signers: [...(await aliceSign(alice)), payerSigner],
-    submitterAccount: payerSigner.id,
-  })
+  const issued = await Issuer.issue(credential, alice as any)
```

Comment:
I’m used to seeing this test as an example of real-life usage of the SDK. As an SDK user I won’t be happy to write `as any` in my code. Is this an indication of an issue? (pun intended)

Reply:
ah, probably a leftover, I don't think it's necessary

Reply:
But more generally, what do you think about the overall approach to interacting with identity primitives here? Is this something you would have liked to implement sporran and SKYC on?

Reply:
I think my judgement is currently clouded by the pain of seeing the bundle-test in its current state. I believe it should be talked over.

Reply:
Yeah, would love to chat, because right now I wouldn't really know what to change here